### PR TITLE
Add e toro

### PR DESCRIPTION
**Pull request by MikeHathaway**

### CODE REVIEW

File: market_maker_keeper/etoro_market_maker_keeper.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+class EToroMarketMakerKeeper:
+    """Keeper acting as a market maker on eToro."""
+
+    logger = logging.getLogger()
+
+    def __init__(self, args: list):
+        parser = argparse.ArgumentParser(prog='eToro-market-maker-keeper')
+
+        parser.add_argument("--etoro-api-server", type=str, default="https://7vriaeqd.hft.etorox.com",
```
Comment (EdNoepel) [inline market_maker_keeper/etoro_market_maker_keeper.py]: Skeptical that this URL should be in a public repo...how are these API URLs generated?

File: bin/etoro-market-maker-keeper [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+dir="$(dirname "$0")"/..
+source $dir/_virtualenv/bin/activate || exit
+export PYTHONPATH=$PYTHONPATH:$dir:$dir/lib/pymaker:$dir/lib/pyexchange:$dir/lib/ethgasstation-client:$dir/lib/gdax-client
```
Comment (EdNoepel) [inline bin/etoro-market-maker-keeper]: I don't think we need `gdax-client` in PYTHONPATH here.  I know several other keepers include it unnecessarily.

File: market_maker_keeper/etoro_market_maker_keeper.py [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+        self.price_feed = PriceFeedFactory().create_price_feed(self.arguments)
+        self.spread_feed = create_spread_feed(self.arguments)
+        self.control_feed = create_control_feed(self.arguments)
+        self.order_history_reporter = create_order_history_reporter(self.arguments)
+
+        self.history = History()
+        self.etoro_api = EToroApi(api_server=self.arguments.etoro_api_server,
+                                account=self.arguments.etoro_account,
+                                api_key=self.arguments.etoro_api_key,
+                                secret_key=open(f'{self.arguments.etoro_secret_key}', 'r').read(),
```
Comment (EdNoepel) [inline market_maker_keeper/etoro_market_maker_keeper.py]: If you haven't already done so, please test and make sure that the failure cases of a missing or bad key file, the user is provided with an error which makes the problem obvious.

File: bin/etoro-market-maker-keeper (hunk 2 above)
Comment (MikeHathaway) [reply bin/etoro-market-maker-keeper]: Removed `gdax-client`

File: market_maker_keeper/etoro_market_maker_keeper.py (hunk 3 above)
Comment (MikeHathaway) [reply market_maker_keeper/etoro_market_maker_keeper.py]: As recommended in prior conversation, replaced with argparse

File: market_maker_keeper/etoro_market_maker_keeper.py (hunk 1 above)
Comment (MikeHathaway) [reply market_maker_keeper/etoro_market_maker_keeper.py]: Removed as suggested

**Pull request merged**